### PR TITLE
Feature/new cart icon

### PR DESCRIPTION
**Pull request by jonhni**
Add new style for shopping cart icon and use relative path to utsjekk

new cart icon from Kimia: 
cart

### CODE REVIEW

**Comment by nilsel** [comment]
@jonhni could you add a example of the icon in webmanual.html (under icons somewhere)?

**Comment by nilsel** [comment]
Also, <h6 class='h2Number'> can cause some confusion for accessibility (many navigate using header lists), can you switch to a span instead?

File: src/js/stb-page-padding.js [hunk 1]
```diff
@@ -609,9 +609,28 @@ var idValidatorCorporate = {
...
+  var env = subHost.split('-')[1] || '';
+  var getOptionalEnv = function() {
+    return env ? '-' + env : '';
+  }
+  var getOrderClientHost = function() {
+    return location.hostname.indexOf('.storebrand.no') > 0
+      ? 'www' + getOptionalEnv() + '.storebrand.no'
+      : 'client' + getOptionalEnv() + '.stb.intra';
+  }
+  return location.protocol + '://' + getOrderClientHost() + path;
```
Comment (estien) [inline src/js/stb-page-padding.js]: Just messing with you 😎  Everyone, this was my bad. 🙈

**Comment by nilsel** [comment]
looking good! new version 0.97 on its way now

**Pull request merged**